modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -27,6 +27,38 @@ This illustration summarizes how each asset type (in green) appears in Exchange:
 
 image::ex2-exchange-assets.png[Illustration - How Assets appear in Exchange]
 
+== Asset Limits
+
+Exchange prevents resource exhaustion attacks by limiting the number of asset versions that can be published.
+
+The limit is 500 assets for master organizations with trial accounts, and 100 000 (one hundred thousand) assets for other master organizations.
+
+This asset count does not include deleted assets or generated assets such as xref:to-deploy-using-rest-connect.adoc[Mule 3 and Mule 4 connectors generated automatically from APIs].
+
+When a master organization has reached 80% of its limit, Exchange shows a warning.
+
+When a master organization has reached its limit, Exchange shows an error. Also, using the *New asset* button on the home page or the *Add new version* button on the asset detail page shows an error explaining that the limit has been reached and no more assets can be added.
+
+Check the limit of your master organization with a CURL command like this. Replace `MASTER_ORGANIZATION_ID` with your master organization ID, and replace `ANYPOINT_TOKEN` with the authorization token with permissions for the master organization.
```

Comment:
Suggest ending the first sentence with a colon, presenting the syntax, and then continuing with the second sentence:

"...like this:

     curl -X GET \...

Replace...."

Reply:
Changed to this style.